the tests parse the tool output and expect one of the strings to match-
samurai outputs slightly different strings, and hence breaks the tests
--- a/unittests/baseplatformtests.py
+++ b/unittests/baseplatformtests.py
@@ -82,5 +82,5 @@
         self.orig_env = os.environ.copy()
         if self.backend is Backend.ninja:
-            self.no_rebuild_stdout = ['ninja: no work to do.', 'samu: nothing to do']
+            self.no_rebuild_stdout = ['ninja: no work to do.', 'samu: nothing to do', 'samu: explain all: phony and no inputs']
         else:
             # VS doesn't have a stable output when no changes are done

